src/Microsoft.Extensions.DiagnosticAdapter/DiagnosticListenerExtensions.cs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.Extensions.DiagnosticAdapter;
+
+namespace System.Diagnostics.Tracing
```

Comment:
@rynowak Actually I think this should be `System.Diagnostics` instead right? That's the namespace for `DiagnosticListener`.